concepts/admission-webhook-cert-management/README.md

```diff
@@ -0,0 +1,145 @@
+# Admission webhook certificate management
+
+This document proposes several ways to manage TLS certificates for custom admission webhooks. 
+
+## Problem
+
+Kubernetes admission controllers act as gatekeepers intercepting API requests and can change the request object or deny its entry to the cluster.
+It is possible to extend the compiled-in admission controllers with custom webhooks using the [dynamic admission control mechanism](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/).
+
+<img src="assets/admission-controller-certs.drawio.svg">
+
+## How Kyma webhooks manage cerificates
+
+A handful of Kyma operators are deployed with custom validating admission webhooks (api-gateway, pod-preset, serverless, telemetry, etc.). They use different approaches to manage certificates, which can be categorized into 2 groups.
```

Comment:
If they refer to the respective Kyma components, we should capitalize the words API Gateway etc.
However, I'm not so sure about pod-preset, so ...?!?? 😕

Reply:
Hmm, in this case, I refer to Kyma operators/webhooks, which do not necessarily have a 1:1 relation to components. pod-preset webhook is a part of cluster essentials

Reply:
Alright! Thanks for explaining